src/scoped.component.js

```diff
@@ -20,11 +20,8 @@ export class Scoped extends React.Component {
     if (!props.css && !props.postcss) throw Error(`Kremling's <Scoped /> component requires either the 'css' or 'postcss' props.`);
     if (props.css && props.postcss) throw Error(`Kremling's <Scoped /> component requires either the 'css' or 'postcss' props. Cannot use both.`);
     if (props.postcss && !(typeof props.postcss.styles === 'string' && props.postcss.id)) throw Error(`Kremlings's <Scoped /> component 'postcss' prop requires an object containing 'styles' and 'id' properties. Try using the kremling-loader.`);
-    if (props.css) {
-      this.state = this.newCssState(props);
-    } else {
-      this.state = this.newPostcssState(props);
-    }
+
+    this.state = this.newCssState(props)
```

Comment:
I combined the `css` and `postcss` props and logic. You can now use the `css` prop with postcss objects. And there aren't two separate codepaths to handle it.

Reply:
yesss! 🎉